terraform/modules/cloudtrail/main.tf

```diff
@@ -9,11 +9,17 @@ module "cloudtrail-cloudwatch" {
   retention_in_days = 731                               // As per RE convention on keeping logs for 2 years
 }
 
+module "cloudtrail-validate-logs-role" {
+  source          = "./modules/cloudtrail-validate-logs-role"
+  assume_role_arn = "${var.validate_account_id}"
```

Comment:
hard to know what `assume_role_arn` is at a glance. It's called `account_id`, `validate_account_id` and `assume_role_arn` in different places. Can we normalise this?